packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -846,9 +846,14 @@ abstract class SliderComponentShape {
 ///  * [SliderTrackShape] Base component for creating other custom track
 ///    shapes.
 class RectangularSliderTrackShape extends SliderTrackShape {
-  /// Abstract const constructor. This constructor enables subclasses to provide
-  /// const constructors so that they can be used in const expressions.
-  const RectangularSliderTrackShape();
+  /// Create a slider track that draws 2 rectangles.
+  const RectangularSliderTrackShape({this.disabledThumbGap = 2.0});
+
+  /// Horizontal spacing, or gap, between the disabled thumb and the track.
+  ///
+  /// This is only used when the thumb is disabled. The Material spec defaults
```

Comment:
It's when the slider is disabled (you can't just disable the thumb, right)?

Reply:
Correct, fixed